src/pages/workspace/reimburse/WorkspaceReimburseView.js

```diff
@@ -64,140 +65,142 @@ const defaultProps = {
-                    icon={Illustrations.MoneyReceipts}
-                    menuItems={[
-                        {
-                            title: this.props.translate('workspace.reimburse.viewAllReceipts'),
-                            onPress: () => Link.openOldDotLink(viewAllReceiptsUrl),
-                            icon: Expensicons.Receipt,
-                            shouldShowRightIcon: true,
-                            iconRight: Expensicons.NewWindow,
-                            wrapperStyle: [styles.cardMenuItem],
-                            link: () => Link.buildOldDotURL(viewAllReceiptsUrl),
-                        },
-                    ]}
-                >
-                    <View style={[styles.mv3, styles.flexRow, styles.flexWrap]}>
-                        <Text numberOfLines={100}>
-                            {this.props.translate('workspace.reimburse.captureNoVBACopyBeforeEmail')}
-                            <CopyTextToClipboard
-                                text="receipts@expensify.com"
-                                textStyles={[styles.textBlue]}
-                            />
-                            <Text>{this.props.translate('workspace.reimburse.captureNoVBACopyAfterEmail')}</Text>
-                        </Text>
-                    </View>
-                </Section>
-
-                <Section
-                    title={this.props.translate('workspace.reimburse.trackDistance')}
-                    icon={Illustrations.TrackShoe}
-                >
-                    <View style={[styles.mv3]}>
-                        <Text>{this.props.translate('workspace.reimburse.trackDistanceCopy')}</Text>
-                    </View>
-                    <OfflineWithFeedback
-                        pendingAction={lodashGet(distanceCustomUnit, 'pendingAction') || lodashGet(distanceCustomRate, 'pendingAction')}
-                        shouldShowErrorMessages={false}
-                    >
-                        <MenuItemWithTopDescription
-                            title={this.state.currentRatePerUnit}
-                            description={this.props.translate('workspace.reimburse.trackDistanceRate')}
-                            shouldShowRightIcon
-                            onPress={() => Navigation.navigate(ROUTES.getWorkspaceRateAndUnitRoute(this.props.policy.id))}
-                            wrapperStyle={[styles.mhn5, styles.wAuto]}
-                            brickRoadIndicator={(lodashGet(distanceCustomUnit, 'errors') || lodashGet(distanceCustomRate, 'errors')) && CONST.BRICK_ROAD_INDICATOR_STATUS.ERROR}
+        Policy.openWorkspaceReimburseView(propsPolicyId);
+    }, [props.policy.id]);
+
+    useEffect(() => {
+        fetchData();
+    }, [fetchData]);
+
+    useEffect(() => {
+        setCurrentRatePerUnit(getCurrentRatePerUnitLabel());
+        const isReconnecting = prevIsOffline && !props.network.isOffline;
+        if (!isReconnecting) {
```

Comment:
Do we need to check againts the previous value here?

Can't we just return early when it's offline?

```suggestion
        if (props.network.isOffline) {
```

Reply:
Yes we can do this as well. I have made the required changes.